kafka-init/src/test/java/io/strimzi/kafka/init/InitWriterTest.java

```diff
@@ -68,7 +68,7 @@ public void testWriteRackId() throws IOException {
 
         InitWriterConfig config = InitWriterConfig.fromMap(envVars);
 
-        KubernetesClient client = mockKubernetesClient(config.getNodeName(), labels, Collections.EMPTY_LIST);
+        KubernetesClient client = mockKubernetesClient(config.getNodeName(), labels, Collections.emptyList());
```

Comment:
I think we normally prefer the use of `List.of()` / `Map.of()`. So unless there is some reason that cannot be used, maybe you can use that instead of `emptyList()` / `emptyMap()`?